numpy/core/numeric.py

```diff
@@ -483,10 +483,7 @@ def asarray(a, dtype=None, order=None):
     See Also
     --------
     asanyarray : Similar function which passes through subclasses.
-    ascontiguousarray : Convert input to a contiguous array.
     asfarray : Convert input to a floating point ndarray.
-    asfortranarray : Convert input to an ndarray with column-major
-                     memory order.
```

Comment:
Do not remove them until they are actually deprecated

Reply:
I'm not sure what you mean by "actually deprecated" - to me this pr _is_ the actual deprecation, and you're referring to the final removal. Perhaps my terminology is wrong.

Reply:
Sorry, I meant actual removal

Reply:
Should I leave all current mentions in the documentation?